cpp/include/ray/api.h

```diff
@@ -162,6 +162,11 @@ PlacementGroup GetPlacementGroup(const std::string &name);
 /// Returns true if the current actor was restarted, otherwise false.
 bool WasCurrentActorRestarted();
 
+/// Get the namespace of this job.
```

Comment:
```suggestion
/// Get the namespace of current job.
```

Reply:
This is copied from https://github.com/ray-project/ray/blob/master/java/api/src/main/java/io/ray/api/runtimecontext/RuntimeContext.java#L43 and they two don't really have difference, so I'm not gonna change it.